src/management/index.js

```diff
@@ -1578,7 +1583,11 @@ utils.wrapPropertyMethod(ManagementClient, 'unblockUser', 'userBlocks.delete');
  *
  * @return  {Promise|undefined}
  */
-utils.wrapPropertyMethod(ManagementClient, 'getUserBlocksByIdentifier', 'userBlocks.getByIdentifier');
+utils.wrapPropertyMethod(
+  ManagementClient,
```

Comment:
I did not change these lines at all however they seemed to automatically format when I checked in my code (the commit did not include these changes until I actually committed)